loleaflet/src/control/Permission.js

```diff
@@ -81,8 +81,10 @@ L.Map.include({
 			this.fire('editorgotfocus');
 			// In the iOS/android app, just clicking the mobile-edit-button is
 			// not reason enough to pop up the on-screen keyboard.
-			if (!(window.ThisIsTheiOSApp || window.ThisIsTheAndroidApp))
-				this.focus();
+			if (!(window.ThisIsTheiOSApp || window.ThisIsTheAndroidApp)) {
+				this.focus(true);
+				this._docLayer.postKeyboardEvent('input', 0, 769); //F2
+			}
```

Comment:
This is great for Calc, but we probably don't want (or need) to send F2 for Writer and Impress.